test/functional/interface_rest.py

```
@@ -64,7 +64,7 @@ def test_rest_request(
             body: str = '',
             status: int = 200,
             ret_type: RetType = RetType.JSON,
-            query_params: typing.Dict[str, typing.Any] = None,
+            query_params: dict[str, typing.Any] = None,
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)